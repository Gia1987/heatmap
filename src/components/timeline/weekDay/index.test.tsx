import { render } from "@testing-library/react";
import WeekDay from "./";

const Component = (props = { index: 3 }) => {
  return <WeekDay {...props} />;
};

describe("<WeekDay/>", () => {
  it("render Wed)", () => {
    const { container } = render(Component());

    expect(container).toMatchSnapshot();
  });
});
