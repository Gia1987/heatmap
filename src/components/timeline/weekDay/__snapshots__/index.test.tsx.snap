// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeekDay/> render Wed) 1`] = `
<div>
  <div
    class="timeline-weekdays-weekday"
  >
    Wed
  </div>
</div>
`;
